### PR TITLE
Add L3X Config

### PR DESCRIPTION
**Pull request by KodeyThomas**
Adds L3X Config as it's an arbitrum based chain most of the config is taken from the existing Arbitrum config.

These values have been tested through ping-pong tests + soak and are working as expected.

### CODE REVIEW

File: core/chains/evm/config/toml/defaults/L3X_Mainnet.toml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+OCR.ContractConfirmations = 1
+LogPollInterval = '10s'
+
+[GasEstimator]
+Mode = 'Arbitrum'
+LimitMax = 1_000_000_000
+# Arbitrum-based chains uses the suggested gas price, so we don't want to place any limits on the minimum
+PriceMin = '0'
+PriceDefault = '0.1 gwei'
+PriceMax = '115792089237316195423570985008687907853269984665.640564039457584007913129639935 tether'
```
Comment (dimriou) [inline core/chains/evm/config/toml/defaults/L3X_Mainnet.toml]: This is already the default value

File: core/chains/evm/config/toml/defaults/L3X_Mainnet.toml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+FeeCapDefault = '1000 gwei'
+BumpThreshold = 5
+
+[GasEstimator.BlockHistory]
+# Force an error if someone set GAS_UPDATER_ENABLED=true by accident; we never want to run the block history estimator on arbitrum
```
Comment (dimriou) [inline core/chains/evm/config/toml/defaults/L3X_Mainnet.toml]: AFAIK `GAS_UPDATER_ENABLED` has been deprecated a long time ago. I don't think this check is relevant anymore.

File: core/chains/evm/config/toml/defaults/L3X_Mainnet.toml [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+FinalityTagEnabled = true
+FinalityDepth = 10
+LinkContractAddress = '0x79f531a3D07214304F259DC28c7191513223bcf3'
+# Produces blocks on-demand
+NoNewHeadsThreshold = '0'
+OCR.ContractConfirmations = 1
+LogPollInterval = '10s'
+
+[GasEstimator]
+Mode = 'Arbitrum'
```
Comment (dimriou) [inline core/chains/evm/config/toml/defaults/L3X_Mainnet.toml:12]: Have we tested whether the existing precompiles work or not?

File: core/chains/evm/config/toml/defaults/L3X_Mainnet.toml [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+LimitMax = 1_000_000_000
+# Arbitrum-based chains uses the suggested gas price, so we don't want to place any limits on the minimum
+PriceMin = '0'
+PriceDefault = '0.1 gwei'
+FeeCapDefault = '1000 gwei'
```
Comment (dimriou) [inline core/chains/evm/config/toml/defaults/L3X_Mainnet.toml:17]: AFAIK both of these are only used in FixedPrice or BlockHistory estimators, so I'm not sure if they're relevant.

File: core/chains/evm/config/toml/defaults/L3X_Mainnet.toml (hunk 3 above)
Comment (KodeyThomas) [reply core/chains/evm/config/toml/defaults/L3X_Mainnet.toml:12]: yes, we've tested this. called the RPC with the precompile address with the correct 4 bytes

File: core/chains/evm/config/toml/defaults/L3X_Mainnet.toml (hunk 4 above)
Comment (KodeyThomas) [reply core/chains/evm/config/toml/defaults/L3X_Mainnet.toml:17]: happy to remove these if needed, most of these values were copied over from the arbitrum config so wanted to ensure the behaviour was identical

**Review by jmank88 — APPROVED** [review]

**Review by dimriou — APPROVED** [review]

**Pull request merged**